internal/server/utils/redisCmdAdapter_test.go

```diff
@@ -26,6 +26,14 @@ func TestParseHTTPRequest(t *testing.T) {
 			expectedCmd:  "SET",
 			expectedArgs: []string{"k1", "v1", "nx"},
 		},
+		{
```

Comment:
Can we add more tests for this?
Have arrays and submaps inside value

Reply:
Added